src/grafana_dashboards/minio-overview_rev13.json.tmpl

```diff
@@ -768,86 +768,6 @@
       "title": "Total Online Servers",
       "type": "stat"
     },
-    {
-      "cacheTimeout": null,
-      "datasource": "${prometheusds}",
-      "description": "",
-      "fieldConfig": {
-        "defaults": {
-          "mappings": [
-            {
-              "options": {
-                "match": "null",
-                "result": {
-                  "text": "N/A"
-                }
-              },
-              "type": "special"
-            }
-          ],
-          "thresholds": {
-            "mode": "absolute",
-            "steps": [
-              {
-                "color": "green",
-                "value": null
-              },
-              {
-                "color": "red",
-                "value": 80
-              }
-            ]
-          },
-          "unit": "short"
-        },
-        "overrides": []
-      },
-      "gridPos": {
-        "h": 2,
-        "w": 3,
-        "x": 3,
-        "y": 6
-      },
-      "id": 9,
-      "interval": null,
-      "links": [],
-      "maxDataPoints": 100,
-      "options": {
-        "colorMode": "value",
-        "graphMode": "area",
-        "justifyMode": "auto",
-        "orientation": "auto",
-        "reduceOptions": {
-          "calcs": [
-            "mean"
-          ],
-          "fields": "",
-          "values": false
-        },
-        "text": {},
-        "textMode": "auto"
-      },
-      "pluginVersion": "8.0.6",
-      "targets": [
-        {
-          "exemplar": true,
-          "expr": "minio_cluster_disk_online_total{job=\"$scrape_jobs\"}",
-          "format": "table",
-          "hide": false,
-          "instant": true,
-          "interval": "",
-          "intervalFactor": 1,
-          "legendFormat": "Total online disks in MinIO Cluster",
-          "metric": "process_start_time_seconds",
-          "refId": "A",
-          "step": 60
-        }
-      ],
-      "timeFrom": null,
-      "timeShift": null,
-      "title": "Total Online Disks",
-      "type": "stat"
-    },
```

Comment:
instead of removing this, can we replace it with `minio_cluster_drive_online_total` from the [Cluster drive metrics](https://min.io/docs/minio/kubernetes/upstream/operations/monitoring/metrics-and-alerts.html#id7)? I think it's equivalent

Reply:
Very good catch, updated!

Reply:
Discussing with Noha, we decided to revert this and remove those disk metrics after all since in our case MinIO doesn't provide them.
Looking at this issue [comment](https://github.com/canonical/bundle-kubeflow/issues/855#issuecomment-2029573498), where metrics from each application at a certain time are exposed, `minio.txt` doesn't include those metrics. In general, we can expect applications to output more metrics than those there, but I guess in this case and since this is a generic metric, this won't happen.